sherpa.spec

```diff
@@ -5,6 +5,10 @@
 Source: git+https://github.com/%github_user/%{n}.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}-%{tag}.tgz
 Requires: hepmc lhapdf blackhat sqlite fastjet openssl
 
+%if "%(case %cmsplatf in (*amd64*) echo true ;; (*) echo false ;; esac)" == "true"
```

Comment:
`*_amd64_*` is more correct approach IMHO. This will not work on AArch64 until compiler and linker improve. This should build on any amd64 target, doesn't matter what linux distribution (for example at some point I was building CMSSW on Fedora 20, ie f20_amd64_*). There is nothing SLC specific here.